BTDB/EventStoreLayer/TypeDescriptorExtensions.cs

```diff
@@ -84,14 +84,19 @@ public static StringBuilder AppendJsonLike(this StringBuilder sb, object obj)
             {
                 return sb.Append("null");
             }
-            var str = obj as string;
-            if (str != null)
-            {
-                return sb.Append('"').Append(str).Append('"');
-            }
+
             // ReSharper disable RedundantToStringCall it is speed optimization
-            return sb.Append(obj.ToString());
+            var objString = obj.ToString();
             // ReSharper restore RedundantToStringCall
+
+            if (obj is string)
```

Comment:
Můžeš napsat `if (obj is string objString)` a zbavíš se těch 3 řádků výš.

Reply:
Tak jednoduchy to nebylo viz muj dalsi commit.